### PR TITLE
Update ingestion.mdx

### PR DESCRIPTION
**Pull request by jcx120**
Removing hardcoded references to stale HW specs (that were confusing customers) and linking them to the Prerequisites doc where we will centralize latest HW requirement specs.

### CODE REVIEW

File: content/docs/run-api-server/ingestion.mdx [hunk 1]
```diff
@@ -111,15 +111,15 @@ In some (albeit rare) cases, it can be convenient to (re)ingest the full Stellar
...
+ 
+  1. For reingestion, the more parallel workers are provisioned to speed up the process, the larger the machine size is required in terms of RAM, CPU, IOPS and disk size. The size of the RAM per worker also increases over time (14GB RAM / worker as of mid 2022) due to the growth of the ledger. HW spec can be downsized once Re-ingestion is completed. 
 
-  1. An instance with 32 CPU cores, 64 GB of RAM and at least 200 GB of disk capacity from which to run Horizon (for example AWS `m5.8xlarge`). This is needed to fit 20 Horizon parallel workers (each with their own Captive Core instance). Each Core instance can take up to 3GB of RAM and a full CPU core (more on why 20 workers below). If the number of workers is increased, you may need a larger machine.
+  2. [Horizon](./installing.mdx) version 2.1x or newer installed on the machine from (1).
 
-  2. [Horizon](./installing.mdx) version 1.6.0 or newer (ideally, 2.x) installed on the machine from (1).
+  3. [Core](https://github.com/stellar/stellar-core) version 18.5 or newer  installed on the machine from (1).
```
Comment (Shaptic) [inline content/docs/run-api-server/ingestion.mdx:120]: Should we just stop mentioning a version here and just say "the latest version of stellar-core" and link to https://github.com/stellar/stellar-core/releases/latest?

File: content/docs/run-api-server/ingestion.mdx [hunk 2]
```diff
@@ -111,15 +111,15 @@ In some (albeit rare) cases, it can be convenient to (re)ingest the full Stellar
 The following instructions assume the reingestion is done on AWS. However, they should be applicable to any other environment with equivalent capacity. In the same way, the instructions can be adapted to reingest only specific parts of the history.
 
 ### Prerequisites
-Before we begin, we make some environment assumptions:
+Before we begin, we make some assumptions around the environment required. Please refer to the [Prerequisites](./prerequisites.mdx) section for the current HW requirements to run Horizon reingestion (for either historical catch up) or real-time ingestion (for staying in synch with the ledger). A few things to keep in mind:
```
Comment (Shaptic) [inline content/docs/run-api-server/ingestion.mdx:114]: mix-up of parentheticals here, I think
```suggestion
Before we begin, we make some assumptions around the environment required. Please refer to the [Prerequisites](./prerequisites.mdx) section for the current HW requirements to run Horizon reingestion for either historical catch up or real-time ingestion (for staying in sync with the ledger). A few things to keep in mind:
```

File: content/docs/run-api-server/ingestion.mdx (hunk 1 above)
Comment (Shaptic) [reply content/docs/run-api-server/ingestion.mdx:120]: And likewise for Horizon?

File: content/docs/run-api-server/ingestion.mdx [hunk 3]
```diff
@@ -111,15 +111,15 @@ In some (albeit rare) cases, it can be convenient to (re)ingest the full Stellar
...
+  2. [Horizon](./installing.mdx) version 2.1x or newer installed on the machine from (1).
 
-  2. [Horizon](./installing.mdx) version 1.6.0 or newer (ideally, 2.x) installed on the machine from (1).
+  3. [Core](https://github.com/stellar/stellar-core) version 18.5 or newer  installed on the machine from (1).
 
-  3. [Core](https://github.com/stellar/stellar-core) version 17 or newer  installed on the machine from (1), given the network is on Protocol 17.
-
-  4. A Horizon database where to reingest the history. Preferably, the database should have at least 32 CPUs, 256 GB of RAM and at least 15K IOPS (for example AWS RDS `r5.8xlarge`) and should be empty to minimize storage (Postgres accumulates data during usage, which is only deleted when `VACUUM`ed).
+  4. A Horizon database where to reingest the history. Preferably, the database should be empty to minimize storage (Postgres accumulates data during usage, which is only deleted when `VACUUM`ed) and have the minimum spec's for reingestion as outlined in [Prerequisites](./prerequisites.mdx).
```
Comment (Shaptic) [inline content/docs/run-api-server/ingestion.mdx:122]: nit: `specs` isn't possessive 

```suggestion
  4. A Horizon database where to reingest the history. Preferably, the database should be empty to minimize storage (Postgres accumulates data during usage, which is only deleted when `VACUUM`ed) and have the minimum specs for reingestion as outlined in [Prerequisites](./prerequisites.mdx).
```

File: content/docs/run-api-server/ingestion.mdx [hunk 4]
```diff
@@ -111,15 +111,15 @@ In some (albeit rare) cases, it can be convenient to (re)ingest the full Stellar
 The following instructions assume the reingestion is done on AWS. However, they should be applicable to any other environment with equivalent capacity. In the same way, the instructions can be adapted to reingest only specific parts of the history.
 
 ### Prerequisites
-Before we begin, we make some environment assumptions:
+Before we begin, we make some assumptions around the environment required. Please refer to the [Prerequisites](./prerequisites.mdx) section for the current HW requirements to run Horizon reingestion (for either historical catch up) or real-time ingestion (for staying in synch with the ledger). A few things to keep in mind:
+ 
+  1. For reingestion, the more parallel workers are provisioned to speed up the process, the larger the machine size is required in terms of RAM, CPU, IOPS and disk size. The size of the RAM per worker also increases over time (14GB RAM / worker as of mid 2022) due to the growth of the ledger. HW spec can be downsized once Re-ingestion is completed.
```
Comment (Shaptic) [inline content/docs/run-api-server/ingestion.mdx:116]: nit: consistency 

```suggestion
  1. For reingestion, the more parallel workers are provisioned to speed up the process, the larger the machine size is required in terms of RAM, CPU, IOPS and disk size. The size of the RAM per worker also increases over time (14GB RAM / worker as of mid 2022) due to the growth of the ledger. HW specs can be downsized once reingestion is completed. 
```

**Review by Shaptic — APPROVED** [review]
tiny notes, good idea overall though :+1:

File: content/docs/run-api-server/ingestion.mdx (hunk 1 above)
Comment (jcx120) [reply content/docs/run-api-server/ingestion.mdx:120]: Yes good idea!

**Pull request merged**